docs/testing_in_vagrant.rst

```diff
@@ -18,12 +18,18 @@ Running CSIT locally in Vagrant
 3. Install vagrant plugins
    From command line run:
       vagrant plugin install vagrant-vbguest
+      
       vagrant plugin install vagrant-cachier
 
    If you are behind a proxy, install proxyconf plugin and update proxy
    settings in Vagrantfile:
       vagrant plugin install vagrant-proxyconf
 
+4. add csit box
```

Comment:
What is the purpose of running ubuntu14? The support in CSIT was dropped some time ago.